extensions/ql-vscode/src/remote-queries/variant-analysis-manager.ts

```diff
@@ -31,6 +32,8 @@ export class VariantAnalysisManager extends DisposableObject implements VariantA
   private readonly variantAnalysisResultsManager: VariantAnalysisResultsManager;
   private readonly variantAnalyses = new Map<number, VariantAnalysis>();
   private readonly views = new Map<number, VariantAnalysisView>();
+  private static readonly maxConcurrentTasks = 3;
```

Comment:
Is the intention that this queue handles more than just downloads? I wasn't expecting so but the generic naming implies this could be case.

I'd suggest rename to `maxConcurrentDownloads` and `downloadQueue` to make it more specific and clear what the fields are for.

I doubt we'll ever want to reuse these for more generic tasks, but if so we can rename the fields again at that point.

Reply:
Renamed: https://github.com/github/vscode-codeql/pull/1599/commits/f2fe1063d93aa41d5eb7d23d61d1edd99f677aae